tests/e2e/postgresql/70-envoy-disabled-install.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: vshn.appcat.vshn.io/v1
```

Comment:
These e2e tests are getting longer and longer. I would suggest to reuse the previous test and disable envoy and test instead of removing and creating it againg. This way you also check the update process of an instance.